package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "dev": "webpack --mode development",
     "build": "webpack --mode production",
-    "start": "webpack --config webpack.config.js --watch"
+    "start": "webpack serve --mode development --open"
```

Comment:
- [ ] Great job applying the change suggested by the previous reviewer, however, I'm getting an error when trying to run `"npm start"`. This happens because `webpack-dev-server` is not listed as a dependency. Please fix this! 🤓 🛠️ 

![image](https://user-images.githubusercontent.com/77069011/211073708-fd848f4f-80e4-486f-a4a6-a5664fe68a65.png)